Makefile

```diff
@@ -41,7 +41,7 @@ _build_check_docker:
 		fi;
 
 .PHONY: gotasks
-gotasks: format lint build test
```

Comment:
Why are we removing the test from here

Reply:
We can't run the bdd test inside gotask because we need chaos-opearator image to run bdd. Which will build after running gotask and dockerops section. so we have to run test cases separately.